packages/styled-components/src/components/primitives/List/Items/ButtonItem/ButtonItem.test.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { render, screen, fireEvent } from '@testing-library/react';
+import { ButtonItem } from '.';
+
+describe('ButtonItem', () => {
+  const testId = 'buttonItem';
+  it('should render correctly with children', () => {
+    render(
+      <ButtonItem>
+        <div>foo</div>
```

Comment:
in other components we only used the text and asserted using getByText

Reply:
Done